docs/development_environment.mdx

```diff
@@ -59,6 +59,24 @@ If the Dev Container was set up correctly - it supports debugging by default, ou
 
 It is also possible to debug a remote Home Assistance instance (e.g., production instance) by following the procedure described [here](https://www.home-assistant.io/integrations/debugpy/).
 
+### Use devcontainer from WSL
+
+With WSL, you won't run into as many path issues because of the '/' and '\' differences between Linux and Windows. 
+This is handy when working on frontend code and using `script/develop`, since it's using stuff that works best on Linux.
+
+**Prerequisites**
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Visual Studio code](https://code.visualstudio.com/)
+- WSL + A WSL Distribution of your choice (Note: The instructions will be based on Ubuntu). Refer to the "Developing on Windows" section below for further instructions and tips.
+
+1. Enable integration with your WSL distro, This can be done in Docker Desktop under Settings > Resources > WSL integration.
+2. Step 1. of "Getting started".
```

Comment:
we should just repeat this single step here, since "go to step 1 of getting started on first section of this document" sounds a bit overwhelming :see_no_evil:

Reply:
You're right.
I also changed step 6.